packages/trader/src/Modules/Trading/Containers/__tests__/chart-widgets.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { ChartBottomWidgets, ChartTopWidgets, DigitsWidget } from '../chart-widgets';
+
+jest.mock('../chart-widgets.tsx', () => ({
+    ...jest.requireActual('../chart-widgets.tsx'),
+    DigitsWidget: () => <div>MockedDigitsWidget</div>,
+    ChartTopWidgets: () => <div>MockedChartTopWidgets</div>,
+    ChartBottomWidgets: () => <div>MockedChartBottomWidgets</div>,
+}));
+
+describe('<DigitsWidget />', () => {
+    it('Should render mocked digits widget', () => {
+        render(<DigitsWidget digits={[0, 1, 2, 3, 4, 5]} tick={{ pip_size: 1 }} />);
```

Comment:
Okay, I not sure, but mocking `<DigitsWidget />` and then testing it is a bit extraordinary